locale/en/blog/community/quality-with-speed.md

```diff
@@ -0,0 +1,316 @@
+# Node.js - Quality with Speed
```

Comment:
I believe there's a metadata header block that can be set at the top to provide author and title information